### ISSUE TITLE
Please update the react-native-windows dependency versions.

### ISSUE THREAD

**Issue by RohitLuthra19** [Microsoft/reactxp#153]
I tried to run the following command 
react-native windows
it gets failed by showing following error message

```
Rohits-MBP:vAdmin rohitluthra$ react-native windows
Scanning 751 folders for symlinks in /Users/rohitluthra/Desktop/vAdmin/node_modules (60ms)
Reading application name from package.json...
Reading react-native version from node_modules...
Checking for react-native-windows version matching 0.42.*...
Installing react-native-windows@0.42.0...
npm WARN react-native@0.42.3 requires a peer of react@~15.4.1 but none was installed.
npm WARN react-native-windows@0.42.0 requires a peer of react@~15.4.1 but none was installed.
npm WARN reactxp@0.42.0-rc.3 requires a peer of react-addons-perf@^15.4.1 but none was installed.
npm WARN reactxp@0.42.0-rc.3 requires a peer of react-native-windows@^0.33.0 but none was installed.
npm ERR! code 1
Command failed: npm install --save react-native-windows@0.42.0
npm WARN react-native@0.42.3 requires a peer of react@~15.4.1 but none was installed.
npm WARN react-native-windows@0.42.0 requires a peer of react@~15.4.1 but none was installed.
npm WARN reactxp@0.42.0-rc.3 requires a peer of react-addons-perf@^15.4.1 but none was installed.
npm WARN reactxp@0.42.0-rc.3 requires a peer of react-native-windows@^0.33.0 but none was installed.
npm ERR! code 1
```

For more detail see https://github.com/Microsoft/react-native-windows/issues/1156

**Comment by johannao76** [comment]
Is there a workaround for this? In my case, I was trying to run npm install and kept getting tripped up by several dependency errors, the last of which was this react-native-windows dependency which I can't seem to be able to workaround.

**Comment by ms-deregtd** [comment]
If you separately just "npm install react-native-windows@0.42.0" does that cause errors?  I don't think that the r-n-w infra plays nicely with the react-native tooling (ios/android only, I believe).

**Comment by johannao76** [comment]
After running this, I still get:
+-- UNMET PEER DEPENDENCY react@15.4.2
`-- UNMET PEER DEPENDENCY react-native-windows@0.42.0
npm WARN reactxp@0.42.0-rc.20 requires a peer of react-native-windows@^0.33.0 but none was installed.
npm WARN react-dom@15.6.1 requires a peer of react@^15.6.1 but none was installed.

**Comment by ms-deregtd** [comment]
As long as they're only warnings, you can ignore them and see if your app compiles and runs properly.  Our npm installs have dozens of unmet peer dependency warnings due to what, as far as we can tell, are bugs in the dependency calc in NPM.

**Comment by johannao76** [comment]
Tried running 'npm run web-watch' under the repo root, and getting this error:

PS E:\reactnative\reactxp> npm run web-watch
npm ERR! Windows_NT 10.0.14393
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "run" "web-watch"
npm ERR! node v7.10.0
npm ERR! npm  v4.2.0

npm ERR! missing script: web-watch

I am trying to run the hello-world sample. Do I need to run all these steps from the hello-world folder instead? I tried that and ran into errors there too.

**Comment by ms-deregtd** [comment]
You should be able to take the hello world example and put it into an entirely separate folder outside of reactxp, delete all of reactxp, and then set it up like an application that uses reactxp from its node_modules directory.  So, if you're just trying to run the sample, don't check out the whole folder just start with the sample and use it from there.

**Comment by johannao76** [comment]
Ok, so I tried that and after running npm install under the new folder get these:
npm WARN react-native@0.42.3 requires a peer of react@~15.4.1 but none was installed.
npm WARN react-native-windows@0.33.4 requires a peer of react-native@~0.33.0 but none was installed.
npm WARN react-native-windows@0.33.4 requires a peer of react@~15.3.1 but none was installed.

I kept going even with these warnings and was able to run the hello-world app in both web (via index.html) as well as Android Studio! Thanks much for your help!
